### PR TITLE
Adds an ided "balanced" dynamic gamemode

### PR DESCRIPTION
**Pull request by ghost**
# Document the changes in your pull request

This PR adds a dynamic subtype, that contains inself mostly solo or non-conversion(in core) antags. Darkspawns and bloodsuckers are still there, because converting dudes as a bloodsucker or darkspawn is only volountary, and they can't convert people in the same amounts as cults can.

It is something like a compromise between fun and interesting gamemodes and balanced gamemodes. If this PR will get merged, i will make another PR that replaces all solo antag gamemodes, that have a dynamic ruleset alternative, with this one.

# Changelog

:cl: SuperSlayer
rscadd: Adds a "balanced" dynamic gamemode, that doesn't contain cults and shadowlings
/:cl:

### CODE REVIEW

File: code/game/gamemodes/dynamic/dynamic.dm [hunk 1]
```diff
@@ -17,7 +17,7 @@ GLOBAL_LIST_EMPTY(dynamic_forced_roundstart_ruleset)
 GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 
 /datum/game_mode/dynamic
-	name = "dynamic mode"
+	name = "Dynamic Mode"
```
Comment (Bibby0110) [inline code/game/gamemodes/dynamic/dynamic.dm:20]: no

File: code/game/gamemodes/dynamic/dynamic.dm [hunk 2]
```diff
@@ -824,3 +826,11 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 #undef FAKE_REPORT_CHANCE
 #undef REPORT_NEG_DIVERGENCE
 #undef REPORT_POS_DIVERGENCE
+
+/datum/game_mode/dynamic/balanced
+	name = "balanced dynamic"
+	config_tag = "balanced_dynamic"
+	report_type = "dynamic"
+	title_icon = "ss13"
```
Comment (Bibby0110) [inline code/game/gamemodes/dynamic/dynamic.dm:834]: no

File: code/game/gamemodes/dynamic/dynamic.dm [hunk 3]
```diff
@@ -824,3 +826,11 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
...
+	announce_span = "danger"
```
Comment (Bibby0110) [inline code/game/gamemodes/dynamic/dynamic.dm:835]: no

File: code/game/gamemodes/dynamic/dynamic.dm [hunk 4]
```diff
@@ -824,3 +826,11 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 #undef FAKE_REPORT_CHANCE
 #undef REPORT_NEG_DIVERGENCE
 #undef REPORT_POS_DIVERGENCE
+
+/datum/game_mode/dynamic/balanced
+	name = "balanced dynamic"
+	config_tag = "balanced_dynamic"
+	report_type = "dynamic"
```
Comment (Bibby0110) [inline code/game/gamemodes/dynamic/dynamic.dm:833]: no

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
variable names are stupid. make them something that implies that its dynamic without team antags

File: code/game/gamemodes/dynamic/dynamic.dm (hunk 4 above)
Comment (ghost) [reply code/game/gamemodes/dynamic/dynamic.dm:833]: ???

**Comment by JamieD1** [comment]
Why we adding a hugbox dynamic

**Comment by ghost** [comment]
> Why we adding a hugbox dynamic

A good idea will be to replace every gamemode with dynamic, but when i asked people about it in development-public chanel they started crying about that "it will be unbalanced" and other shit, so i decided to make a something like a compromise

**Pull request closed without merging**